### PR TITLE
Added a tutorial on how to run "two-mappers" example

### PR DESCRIPTION
**Pull request by tiagofilipe12**
This PR adds a README.md to `examples/pipelines/two-mappers` that can be used as a walk-through guide on how to build a similar pipeline or even to run that pipeline. Also, adds a pipeline_lazy.js that is intended to provide a solution to the proposed challenge in the README.md.

### CODE REVIEW

File: examples/pipelines/two-mappers/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+using bionode-watermill.
+
+## Requirements for this example
+
+First of all, you will need to clone bionode-watermill if you haven't done it
+ yet: `git clone https://github.com/bionode/bionode-watermill`
+
+Also, you need to have installed:
+
+1) **Node.js** version 7 or higher.
```
Comment (thejmazz) [inline examples/pipelines/two-mappers/README.md:15]: I think 6 is sufficient

File: examples/pipelines/two-mappers/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+2) **[bionode-ncbi](https://github.com/bionode/bionode-ncbi)** (however, this 
+one is not really necessary assuming that you 
+run the examples from within the bionode-watermill folders).
+3) **[sra-toolkit](https://www.ncbi.nlm.nih.gov/books/NBK158900/)**
+4) **[bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)**
+5) **[bwa](http://bio-bwa.sourceforge.net/)**
+6) **[samtools](http://samtools.sourceforge.net/)**
+
+However, don't worry you can use [this](https://github.com/tiagofilipe12/docker-watermill-tutorial)
```
Comment (thejmazz) [inline examples/pipelines/two-mappers/README.md:24]: we can publish these on `bionode` organization on docker hub later

**Review by thejmazz — APPROVED** [review]
lgtm

File: examples/pipelines/two-mappers/README.md (hunk 1 above)
Comment (tiagofilipe12) [reply examples/pipelines/two-mappers/README.md:15]: Shall we put 6? I only tested 4 and 7.

**Pull request merged**